build.gradle.kts

```diff
@@ -85,7 +84,6 @@ dependencies {
   integrationTestImplementation(testFixtures(project("domain:goal")))
   integrationTestImplementation(testFixtures(project("domain:timeline")))
   integrationTestImplementation(testFixtures(project("domain:induction")))
-  integrationTestImplementation("org.testcontainers:localstack:1.18.1")
```

Comment:
I've also removed `testcontainers`, since it was only being used to test the SQS functionality. We can always restore it if necessary, but it will be interesting to see if this improves the build time of this API.

Reply:
The results of my "scientific test" following two builds:

- Current main branch: 2 minutes 16 seconds
- This branch: 2 minutes 8 seconds

Disappointing! 😂

Reply:
🤣